views.py

```diff
@@ -610,3 +611,40 @@ def post(self, request, *args, **kwargs):
         return Response(
             serializer(objects.order_by('priority'), many=True).data
         )
+
+class DataLeakView(APIView):
+    """
+    Endpoint to get faculty data using their employeeId
+    """
+
+    permission_classes = (CanDataLeak,)
+    pagination_class = None
+
+    def get(self, request):
+        employee_id = request.GET.get('fac_id')
+
+        if employee_id is None:
+            return Response(
+                { 'Error': ['Faculty id parameter is missing.'] },
+                status=status.HTTP_400_BAD_REQUEST,
+            )
+        try:
+            Faculty = swapper.load_model('kernel', 'facultyMember')
```

Comment:
Change this to `FacultyMember`. I am afraid it will confuse people plus I want to uphold our naming conventions.